modules/extensions-xml-support/src/main/resources/META-INF/mule-module.properties

```diff
@@ -0,0 +1,3 @@
+module.name=extensions-xml-support
+
+artifact.export.classPackages=org.mule.runtime.extension.internal.introspection.describer
```

Comment:
Modules living in the container classloader do not use the filtering, the filtering is only used for other artifacts as apps, domains, services, etc